manifests/opsmanager/install.pp

```diff
@@ -28,8 +28,21 @@
     }
   }
 
+  case $facts['os']['family'] {
+    'RedHat': {
+      $my_provider = 'rpm'
+    }
+    default: {
+      warning("The ${module_name} module might not work on ${facts['os']['family']}.  Sensible defaults will be attempted.")
+      $my_provider = undef
```

Comment:
Do we want to use undef or $facts['package_provider'] here? Does puppet figure out that if provider => undef, just use the default provider?